src/nanoemoji/write_config_for_glyph_svgs.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Generates a glyphmap for svgs named by glyph id."""
+from absl import app
+from absl import logging
+from fontTools import ttLib
+from nanoemoji import config
+from nanoemoji import util
+from pathlib import Path
+import textwrap
+
+
+def main(argv):
+    font_file = util.only(lambda a: a.endswith(".ttf"), argv)
+    config_file = Path(util.only(lambda a: a.endswith(".toml"), argv))
+    font = ttLib.TTFont(font_file)
+    upem = font["head"].unitsPerEm
+    ascender = font["OS/2"].sTypoAscender
+    descender = font["OS/2"].sTypoDescender
```

Comment:
what about `linegap`? it may be != 0, and we should retain that

Reply:
i suppose linegap doesn't really matter since we are discarding the OS/2 table of the intermediate COLR font anyway and only keeping the original one from the OT-SVG font. Ascender/descender on the other hand we need them to correctly scale/position the svgs in the font space.

Reply:
I believe it doesn't matter given the merge back so closing. Plz reopen if I'm missing something I need to copy.